includes/class-experimental-abtest.php

```diff
@@ -88,6 +100,41 @@ public function get_variation( $test_name ) {
 		return $variation;
 	}
 
+
+	/**
+	 * Perform the request for a experiment assignment of a provided A/B test from WP.com.
+	 *
+	 * @param array $args Arguments to pass to the request for A/B test.
+	 * @return array|\WP_Error A/B test variation error on failure.
+	 */
+	public function request_assignment( $args ) {
+		// Request as authenticated wp user.
+		if ( $this->as_auth_wpcom_user && class_exists( Jetpack_Connection_Manager::class ) ) {
```

Comment:
Once issue https://github.com/woocommerce/woocommerce-admin/issues/8460 is resolved, this class will always exist 🎉